tests/unit_tests/test_wave_06.py

```diff
@@ -76,7 +76,10 @@ def test_swap_best_by_category():
         their_priority="Decor"
     )
 
-    raise Exception("Complete this test according to comments below.")
+    # raise Exception("Complete this test according to comments below.")
+    assert result == True
+    assert len(tai.inventory) == 3 and len(jesse.inventory) == 3
+    assert item_c in jesse.inventory and item_f in tai.inventory
```

Comment:
I recommend checking all of the items here to make sure that the exact items we expect are in each inventory:
```suggestion
    assert item_c in jesse.inventory and item_d in jesse.inventory and item_e in jesse.inventory
    assert item_f in tai.inventory and item_a in tai.inventory and item_b in tai.inventory
```